pages/api/studentPortal/quizzes/firstProgrammingLanguage.ts

```diff
@@ -0,0 +1,80 @@
+export type QuizQuestion = {
```

Comment:
this file is called firstProgrammingLanguage, so everything in this file should be specific to that quiz. These custom types can actually be moved into a shared file. maybe we should make a file called `types.ts` and move these shared custom types into that file.

Reply:
@LuckyHariharan make sure these types are moved before you use them in your career quiz

Reply:
The types.ts file lives in the quizzes/shared folder now.